drivers/ieee802154/ieee802154_cc1101.c

```diff
@@ -0,0 +1,873 @@
+/* ieee802154_cc1101.c - TI CC1101 driver */
+
+/*
+ * Copyright (c) 2017 Intel Corporation.
```

Comment:
s/2017/2018
and add another line with your copyright as well, do that on all relevant files you copied/modified